internal/init/init.go

```diff
@@ -135,22 +124,60 @@ func (c *Client) NewCommand() *cobra.Command {
 	return initCmd
 }
 
-func getFirstProjectID(s metal.ProjectsApiService, userOrg string) (string, error) {
-	include := []string{"organization"} // []string | Nested attributes to include. Included objects will return their full attributes. Attribute names can be dotted (up to 3 levels) to included deeply nested objects. (optional)
-	exclude := []string{"devices", "members", "memberships", "invitations", "ssh_keys", "volumes", "backend_transfer_enabled", "updated_at", "customdata", "event_alert_configuration"}
-	resp, err := s.FindProjects(context.Background()).Include(include).Exclude(exclude).ExecuteWithPagination()
+func parseServiceToken(p metal.ProjectsApiService, u metal.UsersApiService) (string, string, error) {
+	var defaultOrgId, defaultProjectId string
+	// Get first page of projects associated with provided token
+	projects, err := getFirstPageOfProjects(p)
 	if err != nil {
-		return "", err
+		return "", "", err
 	}
 
-	projects := resp.Projects
-	for _, p := range projects {
-		if p.Organization.GetId() == userOrg {
-			return p.GetId(), nil
+	switch numProj := len(projects); {
+	case numProj == 0:
+		// If no projects come back, warn the user but assume it's a valid user
+		// token
+		fmt.Println("WARN: No available projects found with the provided API Token")
+		defaultOrgId, defaultProjectId, err = getDefaultIds(u)
+		if err != nil {
+			return "", "", err
 		}
+	case numProj == 1:
+		// If only one project comes back, assume it's a project token
+		// and grab the ProjectID and Org Id from that single project
+		defaultProjectId = projects[0].GetId()
+		fmt.Println("Default Project ID: ", defaultProjectId)
```

Comment:
These `fmt.Println`s are redundant to the prompts which include the defaults..

```
$ metal init
Equinix Metal API Tokens can be obtained through the portal at https://console.equinix.com/profile/api-keys
See https://metal.equinix.com/developers/docs/accounts/users/ for more details.

User Token (hidden): 
Default Project ID:  248...
Default Org ID:  ecd...
Organization ID [ecd...]: 
Project ID [248...]: 

Writing /home/codespace/.config/equinix/metal.yaml
```

Reply:
That was debug output. I've removed it.